src/functionalTest/java/submitBulk/ClaimStatusUpdate.java

```diff
@@ -0,0 +1,65 @@
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import io.restassured.path.xml.XmlPath;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.matchesRegex;
+import static org.hamcrest.CoreMatchers.containsString;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Replace this with `import static org.junit.jupiter.api.Assertions.assertEquals`.  You'll then be able to remove `Assertions.` from before `assertEquals()` on line 34.